src/lib/util/rewriteHTML.ts

```diff
@@ -9,7 +9,7 @@ export const rewriteHTML = (
     result = result.replace(/::baseUriPath::/gi, rewriteValue);
     result = result.replace(/::cdnPrefix::/gi, cdnPrefix || '');
 
-    const faviconPrefix = cdnPrefix ? 'https://cdn.getunleash.io' : '';
+    const faviconPrefix = cdnPrefix ? cdnPrefix : '';
```

Comment:
This could be an oversight... but if we have a cdn prefix I think we should use it. Maybe it was always the same originally but this seems correct

Reply:
Maybe not?
`https://cdn.getunleash.io/v4.1.0/favicon.ico` is not available, but `https://cdn.getunleash.io/favicon.ico` is.

Reply:
Good catch! But maybe it's just a coincidence that that version does not have it. I've checked in our current version and I can find the icon under https://cdn.getunleash.io/unleash/commits/d60ea1acd45e30ddb85e0996dcc24c7bb27e7ff7/favicon.ico on our continuous build. I'll check if it's also present in our recent releases